packages/coil-oauth-scripts/src/Stream.ts

```diff
@@ -319,4 +332,30 @@ export class Stream {
   pause() {
     return this.stop(false)
   }
+
+  // Returns token fraction.
+  private currentStreamSendMax(): number {
+    const elapsed = this.schedule['watch'].time()
```

Comment:
Maybe can expose something public on schedule to get this data with private['investigations'] ?